install_config/master_node_configuration.adoc

```diff
@@ -1148,11 +1148,40 @@ AUDIT: id="5c3b8227-4af9-4322-8a71-542231c3887b" ip="127.0.0.1" method="GET" use
 AUDIT: id="5c3b8227-4af9-4322-8a71-542231c3887b" response="200"
 ----
 
-To enable and configure audit, edit the *_/etc/origin/master/master-config.yaml_*
-file or use a command similar to the following:
+[[proc-master-node-config-audit-config]]
+==== Enable Basic Auditing
 
+The following procedure enables basic auditing post installation.
+
+[NOTE]
+====
+Advanced Audit must be enabled during installation.
+====
+
+. Edit the *_/etc/origin/master/master-config.yaml_* file on all master nodes as shown in the example below:
```

Comment:
```suggestion
. Edit the *_/etc/origin/master/master-config.yaml_* file on all master nodes as shown in the following example:
```

Per the ISG:
> ! below adjective, adverb, preposition
...Do not use to indicate a relative location in a document, as in "the information below".